src/webui/src/components/TrialsDetail.tsx

```diff
@@ -60,55 +60,57 @@ class TrialsDetail extends React.Component<TrialsDetailProps, TrialDetailState>
             tablePageSize: 20,
             whichGraph: '1',
             searchType: 'id',
-            searchFilter: trial => true,
+            // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+            searchFilter: trial => true // eslint-disable-line @typescript-eslint/no-unused-vars
         };
     }
 
     // search a trial by trial No. & trial id
-    searchTrial = (event: React.ChangeEvent<HTMLInputElement>) => {
+    searchTrial = (event: React.ChangeEvent<HTMLInputElement>): void => {
         const targetValue = event.target.value;
-        let filter = (trial: Trial) => true;
+        // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+        let filter = (trial: Trial) => true; // eslint-disable-line @typescript-eslint/no-unused-vars
         if (!targetValue.trim()) {
             this.setState({ searchFilter: filter });
             return;
         }
         switch (this.state.searchType) {
             case 'id':
-                filter = trial => trial.info.id.toUpperCase().includes(targetValue.toUpperCase());
+                filter = (trial): any => trial.info.id.toUpperCase().includes(targetValue.toUpperCase()); // elint-disable-line
```

Comment:
The return type is `boolean`.
*elint*-disable-line does nothing I suppose.

Reply:
Had fixed. Thanks~